servers/physics_server.cpp

```diff
@@ -732,3 +734,43 @@ PhysicsServer::~PhysicsServer() {
 
 	singleton = NULL;
 }
+
+Vector<String> PhysicsServerManager::physics_servers_classes;
```

Comment:
Might be the C++ noob in me talking, but why should it be redeclared here if it's already in the header? Also, the initializations should typically go in a constructor, no?

Reply:
This is a static variable, and this is the way how to declare it